assets/app/view/game/game_log.rb

```diff
@@ -16,6 +16,7 @@ def render
         ]
 
         @player = @game.player_by_id(@user['id']) if @user
+        sender = @player || Engine::Player.new(@game_data['user']['id'], @game_data['user']['name'])
```

Comment:
why is this created outside of the lambda?

Reply:
why does this use a name instead of an id?

Reply:
It is creating the object with both